modules/op-running-unpriveleged-builds-in-a-container.adoc

```diff
@@ -0,0 +1,62 @@
+:_content-type: PROCEDURE
+
+[id='op-running-unpriveleged-builds-in-a-container._{context}']
+= Running unprivileged builds in a container
+
+You can run builds as a non-root user inside a container to restrict specific container activities and enhance security.
+
+[INFO]
+====
+To run as a non-root user inside the container, your container must first add the `subuid` and `subgid` with the latest shadow-utils.
+====
+
+.Procedure
+
+. Use the public https://catalog.redhat.com/software/containers/ubi8/buildah/602686f7b16b1eb2e30807ee?container-tabs=dockerfile[Containerfile] image based on Red Hat UBI.
+
+. Force run the container as the “build” user to prevent it from being “randomed” which will not have the setup needed to run as non-root.
+
+. Add `securityContext`` to each build and push steps to directly edit or export to a specific namespace the buildah ClusterTask.
```

Comment:
I'm not sure how to interpret this sentence. Would you revise it for clarity?

Reply:
```suggestion
. Add `securityContext` to each build and push steps to directly edit or export to a specific namespace the buildah `ClusterTask` resource.
```

Reply:
Please:
- Remove the surplus backtick.
- Either label the `ClusterTask` as a resource or call it a "cluster task."

Reply:
Any `resource` in backticks / where a literal name is used should be followed by a noun explaining what it is. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting in the guidelines. I'd say this also applies for `securityContext`.